assets/bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "assets",
```

Comment:
I know, I was just trying to strictly follow the given instructions. (one main html + everything else in a folder)